src/MOI_wrapper/results.jl

```diff
@@ -40,7 +40,7 @@ function MOI.get(o::Optimizer, ::MOI.ResultCount)::Int
 end
 
 function MOI.get(o::Optimizer, ::MOI.RawStatusString)
-    return String(CEnum.enum_name(SCIPgetStatus(o)))
+    return String(Symbol(SCIPgetStatus(o)))
```

Comment:
This is probably a consequence of upgrading `CEnum`?

Reply:
The error occurs because CEnum does not have an enum_name field.

Reply:
Well, it has in version 0.1.0: [CEnum.jl](https://github.com/JuliaInterop/CEnum.jl/blob/v0.1.0/src/CEnum.jl#L45)